src/components/AngerNoodle/AngerNoodle.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Board from './Board';
+import MessageBox from './MessageBox';
+import Score from './Score';
```

Comment:
I'd recommend creating a file that imports all of angerNoodles components and exports them for this component to grab onto (possibly having a "components" folder within "anglerNoodle/" so you can treat it as its entity). This'll make it much easier to tell apart anglerNoodles entrypoints from it's individual pieces